lms/djangoapps/instructor_task/api.py

```diff
@@ -279,7 +279,10 @@ def submit_bulk_course_email(request, course_key, email_id):
     # We also pull out the targets argument here, so that is displayed in
     # the InstructorTask status.
     email_obj = CourseEmail.objects.get(id=email_id)
-    targets = [target.target_type for target in email_obj.targets.all()]
+    targets = [
+        target.short_display()
+        for target in email_obj.targets.all()
+    ]
```

Comment:
Since you can get those data from the `email_obj`'s `id`, I think it makes more sense just to pass the `email_obj`'s `id` to the task input. Then, when you need these display names, do the query then, in the background task itself. That way you're sure to avoid the 255 char limit. That's at least the pattern we've used for other tasks that have run into this issue.